gocd/templates/bash/check-sentry-errors.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# TODO: Make this work with s4s
+/devinfra/scripts/checks/sentry/release_error_events.py \
+  --project-id="${SENTRY_PROJECT_ID}" \
+  --project-slug="${SENTRY_PROJECT}" \
```

Comment:
Is it possible to check multiple projects at once? PoPs could cause issues in processing relays, and vice versa.

Reply:
Yep! That is definitely possible! I'll update the code.